README.md

```diff
@@ -261,3 +261,18 @@ The guest container is running on a network called `mx-tester-synapse-$(TAG)`,
 where `TAG` is the Docker tag for the version of Synapse running. By default,
 that's `matrixdotorg/synapse:latest`.
 
+# Synapse notes
+
+## Rate limits
+
+Since version 0.3.3, by default, mx-tester expands considerably the rate limits for Synapse. We assume that
+this is the preferred behavior for many tests.
+
+If you do NOT wish to expand the rate limits, you need to override them in the `homeserver` section of `mx-tester.yml`,
+e.g.
+
+```yaml
+homeserver:
+  # ...
+  rc_login: synapse-default # Restores the Synapse default
```

Comment:
This doesn't seem to work with options that require `large_rate_limits` e.g. `rc_message`. if you provide them either directly

```
rc_message: synapse-default
```

or apart

```
rc_message:
  per_second: synapse-default
  burst_count: synapse-default
```

Reply:
Note: I have added `synapse-default` as a followup to [this](https://github.com/matrix-org/mx-tester/pull/33/files/d8f29444d1e8dcdca80fb2b4a266297199175518#diff-b1a35a68f14e696205874893c07fd24fdb88882b47c23cc0e0c80a30c7d53759) comment.

Reply:
yeah, have you got an example mx-tester.yml that shows how you'd restore default limits for `rc_message`? I'm still having the same problem when editing mjolnir's like this

```
homeserver:
  # Basic configuration.
  server_name: localhost:9999
  public_baseurl: http://localhost:9999
  registration_shared_secret: REGISTRATION_SHARED_SECRET
  # Make manual testing easier
  enable_registration: true
  enable_registration_without_verification: true

  # We remove rc_message so we can test rate limiting,
  # but we keep the others because of https://github.com/matrix-org/synapse/issues/11785
  # and we don't want to slow integration tests down.
  rc_message:
    per_second: synapse-default
    burst_count: synapse-default
```

or

```
  rc_message: synapse-default
```

Reply:
Found the error in the test. Found the error in the code while I was there, too :)

Thanks for catching this!

Reply:
Ok, nice it's working now and i have confirmed by double checking the generated homeserver.yaml